sys/include/net/telnet.h

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (C) 2021 ML!PA Consulting GmbH
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    net_telnet basic Telnet server implementation
+ * @ingroup     net_ipv6
+ * @brief       Telnet server functions
+ * @{
+ *
+ * @file
+ * @brief   Telnet server ([RFC 855](https://tools.ietf.org/html/rfc855)) implementation
+ * @note    This implementation only supports a single client
+ *
+ * Telnet is entirely unencrypted! Do not use it on public networks.
+ * This is intended to aid debugging on networks that are isolated from the Internet.
```

Comment:
Not big enough?

Reply:
Not in-code. I meant a `LOG_WARNING()` that is shown at least at server start ;-).

Reply:
But this should stay as well.

Reply:
(and maybe also on connect as a MOTD?)

Reply:
\+ it does not even use the doxygen `@warning` command ;-)